src/DotLiquid/Util/NumericConverter.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace DotLiquid.Util
+{
+    /// <summary>
+    /// ExtensionMethods for converting to numeric values.
+    /// </summary>
+    public static class NumericConverter

```

Comment:
The typical naming convention for a class containing extensions is `<Extended class>Extensions`. This makes the extension methods easier to discover because if you see some code calling an extension method on say a string, you know you will find the implementation in the file StringExtensions.cs

Reply:
This started as StringExtensions, and was renamed when I added objects. It shoukd probably just not use the Extension syntactic sugar.